lib/plugins/Label_H1_FTX.test.ts

```diff
@@ -0,0 +1,73 @@
+import { MessageDecoder } from '../MessageDecoder';
+import { Label_H1_FTX } from './Label_H1_FTX';
+
+describe('Label_H1_FTX', () => {
+    let plugin: Label_H1_FTX;
+
+    beforeEach(() => {
+        const decoder = new MessageDecoder();
+        plugin = new Label_H1_FTX(decoder);
+    });
+
+    test('matches Label H1 Preamble FTX qualifiers', () => {
+        expect(plugin.decode).toBeDefined();
+        expect(plugin.name).toBe('label-h1-ftx');
+        expect(plugin.qualifiers).toBeDefined();
+        expect(plugin.qualifiers()).toEqual({
+          labels: ['H1'],
+          preambles: ['FTX', '- #MDFTX'],
```

Comment:
And the formats are both covered? Ok great.

Reply:
Presumably the same other than the preamble?

Reply:
correct!